platform/stateful-application/backend.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  backend "s3" {
+    # From Makefile
+    # terraform init  -backend-config="bucket=nstask" -backend-config="key=Task/develop"
+    # key    = "nstask/Matillion"
+    region = "us-east-1"
+    dynamodb_table = "terraform_states"
+  }
+}
+
+data "terraform_remote_state" "cloud" {
+  backend = "s3"
+  config = {
+#    bucket  = "terraform-states.nearsoft"
+    bucket  = "alfredo-terraform-states.nearsoft"
```

Comment:
i think we need to use the same statement, i think you change this for your own test, but, i the real project this should be the same state like the others or user envs variables

we can talk about this

Reply:
You are right. This is a typo. I was using a different bucket for testing. Just updated the bucket name to the correct one.